.github/workflows/build.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         with:
           node-version: ${{matrix.node}}
       - name: Setup repo
-        run: npm run setup
+        run: npm install
```

Comment:
Does `nx` build on install?

Reply:
Nope, you can configure it to do it with postinstall script tho